### PR TITLE
Basic setup

### PR DESCRIPTION
**Pull request by Mesi21**
- Add the README.md file and MIT.md license file
- Add linters
- Add person.rb file and construct the basic Person class
- Add teacher.rb file and construct the basic Teacher class
- Add student.rb file and construct the basic Student class

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Student < Person
   attr_accessor :classroom
 
   def initialize(classroom)
+    super()
```
Comment (Kingstalux) [inline student.rb]: - [ ] Good job so far 👍 in the initialize method, kindly add all the parameters needed to create a `student` object which are age, classroom, name = "Unknown", parent_permission = true.
- [ ] also when calling the super(), you've to add the parameters gotten from the parent class. Therefore, you should have something like this:   

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
```
Comment (Kingstalux) [inline teacher.rb]: - [ ] Same issue here. Kindly fix your initialize method parameters and the super 👍

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Mesi21, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966

## Highlights 👏 
✅ Linters are passing 👍 
✅ Good commit history 👏 
✅ Good PR description 👌 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Mesi21, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**